numpy/core/src/multiarray/dtype_transfer.c

```diff
@@ -375,11 +375,11 @@ static NpyAuxData *_align_wrap_data_clone(NpyAuxData *data)
     return (NpyAuxData *)newdata;
 }
 
-static void
-_strided_to_strided_contig_align_wrap(char *dst, npy_intp dst_stride,
+static inline void
+_strided_to_strided_contig_align_wrap_helper(char *dst, npy_intp dst_stride,
                         char *src, npy_intp src_stride,
                         npy_intp N, npy_intp src_itemsize,
-                        NpyAuxData *data)
+                        NpyAuxData *data, int init_dest, int safe)
```

Comment:
I wonder if we can start using `bool` here. Thoughts, @seberg, @mattip?
```suggestion
                        NpyAuxData *data, bool init_dest, bool safe)
```

Reply:
Sure, sounds good to me, this is private anyway. `safe` is a good word here, or should it be `needs_api` or `force_api` or so?

Reply:
`needs_api` or even `uses_api` would probably be a better name.

Reply:
Where is the best place to `#include <stdbool.h>`? I assume it's somewhere outside of this file?

Reply:
Maybe best just to use `npy_bool` for now and `NPY_FALSE`/`NPY_TRUE` - we can switch everything over to real bools at some later date.